### ISSUE TITLE
Problem with running python code

### ISSUE THREAD

**Issue by marianpa** [prashnani/PerceptualImageError#2]
I tried to run the python code and I am getting following errors:

**For Tensorflow variant:**

```
[marian@localhost PerceptualImageError]$ python test_PieAPP_TF.py --ref_path imgs/ref.png --A_path imgs/A.png --sampling_mode sparse
  File "test_PieAPP_TF.py", line 82
    image_ref_batch: im_Ref
                  ^
SyntaxError: invalid syntax
```

**For PyTorch variant:**

```
[marian@localhost PerceptualImageError]$ python test_PieAPP_PT.py --ref_path imgs/ref.png --A_path imgs/A.png --sampling_mode sparse
downloading dataset
--2018-10-23 22:10:58--  https://web.ece.ucsb.edu/~ekta/projects/PieAPPv0.1//weights/PieAPPv0.1.pth
Resolving web.ece.ucsb.edu (web.ece.ucsb.edu)... 128.111.56.99
Connecting to web.ece.ucsb.edu (web.ece.ucsb.edu)|128.111.56.99|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 273509598 (261M)
Saving to: ‘PieAPPv0.1.pth’

PieAPPv0.1.pth                    100%[============================================================>] 260.84M  1.62MB/s    in 2m 3s

2018-10-23 22:13:02 (2.13 MB/s) - ‘PieAPPv0.1.pth’ saved [273509598/273509598]

PieAPPv0.1.pth not downloaded
```

After manually moved the file PieAPPv0.1.pth to weights folder I got following:

```
[marian@localhost PerceptualImageError]$ mv PieAPPv0.1.pth weights/
[marian@localhost PerceptualImageError]$ python test_PieAPP_PT.py --ref_path imgs/ref.png --A_path imgs/A.png --sampling_mode sparse
Traceback (most recent call last):
  File "test_PieAPP_PT.py", line 61, in <module>
    PieAPP_net.load_state_dict(torch.load('weights/PieAPPv0.1.pth'))
  File "/home/ok/.local/lib/python2.7/site-packages/torch/nn/modules/module.py", line 719, in load_state_dict
    self.__class__.__name__, "\n\t".join(error_msgs)))
RuntimeError: Error(s) in loading state_dict for PieAPP:
        size mismatch for ref_score_subtract.weight: copying a param of torch.Size([1, 1]) from checkpoint, where the shape is torch.Size([1]) in current model.
```

The windows executable works well.

**Comment by prashnani** [comment]
Thanks for your interest in PieAPP and for confirming that the windows .exe works well!

I have fixed the tensorflow bug and also the problem with the location of the download script. Could you please confirm?

Regarding the pytorch error, it arises because the models currently available were saved with pytorch version 0.3.1. You might be trying with a more recent pytorch version? 

Loading a model in pytorch is backward compatible but not forward compatible (see [here](https://discuss.pytorch.org/t/loading-pre-trained-bninception-in-pytorch-0-4/21966)). A quick fix is to install an older pytorch using one of the wheel files from [here](https://pytorch.org/get-started/previous-versions/). 

I will update the .pth file to support a more recent pytorch version later this week. But, with pytorch v0.3.1, the test script should work, if you want to try it out right away.

If these fixes are satisfactory, could you please close the issue? Thanks! :)

**Comment by ionvision** [comment]
Hi, I was wondering are you still planning to release a pretrained model compatible with recent PyTorch releases (>=1.0)? Thanks!
